<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PX4Forensic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>611</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>비행 데이터</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>file</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_file">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>261</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>501</width>
        <height>381</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_point">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>441</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>491</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_status">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>441</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>331</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="radio_safepoint">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Safe Point</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_geofencepoint">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>220</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Geofence Point</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radio_waypoint">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>280</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Waypoint</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_connected">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>맑은 고딕</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="dataRefreshButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>데이터 불러오기</string>
       </property>
      </widget>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>frame_4</zorder>
     <zorder>frame</zorder>
     <zorder>frame_5</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>로그 데이터</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>log file</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_file_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_file_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>file</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_file_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>421</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>data</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_file_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>421</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>parameters</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>log messages</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_file_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>420</y>
        <width>421</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>81</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>추출</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>설정 데이터</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_config1">
    <property name="title">
     <string>설정</string>
    </property>
    <widget class="QMenu" name="menu_port2">
     <property name="title">
      <string>포트</string>
     </property>
    </widget>
    <addaction name="menu_port2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>도움말</string>
    </property>
   </widget>
   <addaction name="menu_config1"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPORT1">
   <property name="text">
    <string>PORT1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
